website/content/tutorials/1-getting-started/_index.md

```diff
@@ -0,0 +1,954 @@
+  <short-name>computer</short-name>
+  <namespace>http://example.com/ns/computer</namespace>
+  <json-base-uri>http://example.com/ns/computer</json-base-uri>
+  <define-assembly name="computer"/>
+</METASCHEMA>
+```
+
+The metadata above provides useful information to to us Metaschema developers and ours tools that parse Metaschema definitions. The `schema-name` is the long-form, descriptive name of the computer model. The `schema-version` is to give the model itself a version number, for either developers or their tools to use. The `short-name` is the shortened form of the `schema-name`. Normally, Metaschema-enabled tools will parse or generate data with this name `computer`, not `Computer Model`. The `namespace` is a URI used to identify the model and its parts as belonging to a single scope for XML data and schemas. Similarly, the `json-base-uri` serves a similar purpose for JSON data and schemas.
+
+It is important to note this definition is just a starting point. This definition is the most minimally viable definition possible: it is well-formed and valid against [the XML Schema for the Metaschema syntax itself](), but our Metaschema-enabled tools should consider this an empty definition. We have not yet declared a `root-name` and there is no data model yet, so let's start one. We will begin by desinging a computer object to have just an identifier. 
+
+We will now add to the [`assembly`](/specification/concepts/terminology/#assembly) for a computer itself and give it an identifier `flag`.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<METASCHEMA xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0">
+  <schema-name>Computer Model</schema-name>
+  <schema-version>0.0.2</schema-version>
+  <short-name>computer</short-name>
+  <namespace>http://example.com/ns/computer</namespace>
+  <json-base-uri>http://example.com/ns/computer</json-base-uri>
+  <define-assembly name="computer">
+    <formal-name>Computer Assembly</formal-name>
+    <description>A container object for a computer, its parts, and its sub-parts.</description>
+    <root-name>computer</root-name>
+    <define-flag name="id" as-type="string" required="yes">
+        <formal-name>Computer Identifier</formal-name>
+        <description>An identifier for classifying a unique make and model of computer.</description>
+    </define-flag>
+  </define-assembly>
+</METASCHEMA>
+```
+
+With the definition of the first `assembly` our Metaschema definition takes shape. Firstly, we now add the `root-name` element to identify for Metaschema-enabled tools this `assembly`, whether there are one or more top-level ones defined, is the root element of our data models. This will come in handy later. In the `assembly` declaration, we have defined a `computer` that must have an `id`, and it must the value of it must be a `string`. With Metaschema-enabled tooling, we can have information about a computer like below in their respective JSON, XML, and YAML data formats.
```

Comment:
Would rather say

> this `assembly`, whether there are one or more top-level ones defined, can serve as a root element (object) of our data models.

... unless you think it's too complicated at this point to think about a schema where more than one element can be a root (which we can and do have).

Another way of looking at it is that it's by adding `root-name` that you 'turn on' an assembly for use ("as a document"), along with all its directly and indirectly dependent fields, flags and assemblies (though they can't be roots unless they are also defined with `root-name`). The different names (`root-name` and `use-name`) additionally add flexibility to designers (I know, TMI).

s/and it must the value of it must be/the value of which must be

Reply:
Almost missed this one when double-checking, added.